internal/dump/testdata/elasticsearch-7-apache-dump-all/ingest_pipelines/logs-apache.access-1.3.4.json

```diff
@@ -1,202 +1,202 @@
+                "name": "logs-apache.access-1.3.4-third-party"
+            }
+        },
+        {
+            "set": {
+                "field": "event.ingested",
+                "value": "{{_ingest.timestamp}}"
+            }
+        },
+        {
+            "set": {
+                "field": "ecs.version",
+                "value": "1.12.0"
+            }
+        },
+        {
+            "rename": {
+                "field": "message",
+                "target_field": "event.original"
+            }
+        },
+        {
+            "grok": {
+                "field": "event.original",
+                "patterns": [
+                    "%{IPORHOST:destination.domain} %{IPORHOST:source.ip} - %{DATA:user.name} \\[%{HTTPDATE:apache.access.time}\\] \"(?:%{WORD:http.request.method} %{DATA:_tmp.url_orig} HTTP/%{NUMBER:http.version}|-)?\" %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)( \"%{DATA:http.request.referrer}\")?( \"%{DATA:user_agent.original}\")?",
+                    "%{IPORHOST:source.address} - %{DATA:user.name} \\[%{HTTPDATE:apache.access.time}\\] \"(?:%{WORD:http.request.method} %{DATA:_tmp.url_orig} HTTP/%{NUMBER:http.version}|-)?\" %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)( \"%{DATA:http.request.referrer}\")?( \"%{DATA:user_agent.original}\")?",
+                    "%{IPORHOST:source.address} - %{DATA:user.name} \\[%{HTTPDATE:apache.access.time}\\] \"-\" %{NUMBER:http.response.status_code:long} -",
+                    "\\[%{HTTPDATE:apache.access.time}\\] %{IPORHOST:source.address} %{DATA:apache.access.ssl.protocol} %{DATA:apache.access.ssl.cipher} \"%{WORD:http.request.method} %{DATA:_tmp.url_orig} HTTP/%{NUMBER:http.version}\" (-|%{NUMBER:http.response.body.bytes:long})"
+                ],
+                "ignore_missing": true
+            }
+        },
+        {
+            "uri_parts": {
+                "field": "_tmp.url_orig",
+                "ignore_failure": true
+            }
+        },
+        {
+            "remove": {
+                "field": [
+                    "_tmp"
+                ],
+                "ignore_missing": true
+            }
+        },
+        {
+            "set": {
+                "field": "url.domain",
+                "value": "{{destination.domain}}",
+                "if": "ctx.url?.domain == null \u0026\u0026 ctx.destination?.domain != null"
+            }
+        },
+        {
+            "set": {
+                "field": "event.kind",
+                "value": "event"
+            }
+        },
+        {
+            "set": {
+                "field": "event.category",
+                "value": "web"
+            }
+        },
+        {
+            "set": {
+                "field": "event.outcome",
+                "value": "success",
+                "if": "ctx?.http?.response?.status_code != null \u0026\u0026 ctx.http.response.status_code \u003c 400"
```

Comment:
Take into account that the JSONformatter is replacing invalid HTML characters with the UTF-8 encoding, this is done [by default](https://pkg.go.dev/encoding/json#Marshal) for HTML purpouses, but not really needed here. Previous formatter was just indenting the files for readability, without Marshalling and Unmarshalling them.

In any case I guess it doesn't mind here if we are comparing documents formatted in the same way.

Reply:
I would keep it as is for consistency reasons - the same formatter used everywhere, unless HTML entities are considered to be a bug in this case.

Reply:
Well, thinking a second time about this, this is not only used for testing. We have to take into account that these are dumps from Elasticsearch/Kibana, we are modifying them here, what could be unexpected, a user dumping these objects with `elastic-package` is going to see something different to what is seen in Kibana or using the APIs.

Also, if we modify them, we should also check that they continue being valid and generate the same result if imported back in Elasticsearch/Kibana.

Thinking about this, and the purpouse of the dump command I would suggest to go back to the previous formatting.

Reply:
Ok, I see now the reason there was an explicit formatJSON method previously.
As you mentioned, it should be equivalent. However, as these dump commands are intended to be used as an exploratory tool for users, I think it would be better to just revert this change/commit and use the previous one, so these files can be inspected/debugged easier.